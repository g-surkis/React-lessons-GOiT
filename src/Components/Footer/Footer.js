import React from 'react';
import './Footer.scss';

const Footer = () => (
  <div className="footer">
    <a>Policy |</a>
&nbsp;
    <a>About project |</a>
&nbsp;
    <span>
      &#169;
      {new Date().getFullYear()}
      Lviv
    </span>
  </div>
);

Footer.propTypes = {

};

export default Footer;
