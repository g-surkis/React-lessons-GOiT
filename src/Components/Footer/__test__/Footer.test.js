import React from 'react';
import { render } from '@testing-library/react';
import Footer from '../Footer';

describe('Footer', () => {
  it('Make snapshot of Footer', () => {
    const { asFragment } = render(
      <Footer />,
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
