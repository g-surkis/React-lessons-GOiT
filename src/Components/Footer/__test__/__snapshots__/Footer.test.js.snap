// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Make snapshot of Footer 1`] = `
<DocumentFragment>
  <div
    class="footer"
  >
    <a>
      Policy |
    </a>
     
    <a>
      About project |
    </a>
     
    <span>
      ©2020Lviv, Ukraine
    </span>
  </div>
</DocumentFragment>
`;
